apps/dot-voting/app/utils/vote-styled-components.js

```diff
@@ -1,32 +1,27 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import iconAllocations from '../assets/iconAllocations.svg'
-import iconProjects from '../assets/iconProjects.svg'
-import iconDotVoting from '../assets/iconDotVoting.svg'
 
-const badgeDetails = {
-  'allocation': { text: 'Allocations', icon: iconAllocations },
-  'curation': { text: 'Issue Curation', icon: iconProjects },
-  'informational': { text: 'Informational', icon: iconDotVoting },
-}
-// TODO: remove default type once we now it is arriving, or making a generic type
-export const GenerateBadge = ({ type = 'allocation' }) => (
+export const GenerateBadge = ({ label, iconSrc }) => (
   <div css={`
-    display: inline-flex;
-    font-weight: 300;
-    border-radius: 4px;
-    font-size: 16px;
-    line-height: 1.5;
-    white-space: nowrap;
-    color: #30404F;
-    background-color: rgb(12,196,230,0.09);
-  `}>
-    <img src={badgeDetails[type].icon} alt="" />
-    <span css="margin: 0 10px; padding-top: 3px; height: 24px">{badgeDetails[type].text}</span>
+      display: inline-flex;
+      font-weight: 300;
+      border-radius: 4px;
+      font-size: 16px;
+      line-height: 1.5;
+      white-space: nowrap;
+      color: #30404F;
+      background-color: rgb(12,196,230,0.09);
```

Comment:
We might want to use named colors from aragon-ui here

Reply:
Agreed. However, I noticed that the Voting app uses aragon-ui verions 1.0.0-alpha.23 which contains an AppBadge component. If we switch to that, we can just get rid of vote-styled-component. If that's something we don't want to do, I can update the GenerateBadge component with proper theme variables.

Reply:
I general I am more towards following our internal development conventions and then make the needed changes once we update, or in case you feel we need to update aragon-ui do it yourself, but generally avoid merging things that accumulate tech debt to our repo, that is just my preference, do whatever you think is fair @PeterMPhillips